api/src/main/java/com/capitalone/dashboard/service/DeployServiceImpl.java

```diff
@@ -236,10 +222,7 @@ private CollectorItem createCollectorItem(Collector collector, DeployDataCreateR
         tempCi.setPushed(true);
         tempCi.setLastUpdated(System.currentTimeMillis());
         tempCi.setNiceName(request.getNiceName());
-        Map<String, Object> option = new HashMap<>();
-        option.put("applicationName", request.getAppName());
-        option.put("instanceUrl", request.getInstanceUrl());
-        tempCi.getOptions().putAll(option);
+        tempCi.getOptions().put("applicationName", request.getAppName());
```

Comment:
CI's for difference instances are intentionally different. Removing the instanceURL will break this.